api/src/main/resources/emails/welcomeemail/WelcomeEmail.html

```diff
@@ -0,0 +1,125 @@
+      line-height: 36px;
+    }
+
+    .text {
+      color: rgba(0,0,0,0.87);
+      font-size: 16px;
+      font-weight: 300;
+      line-height: 22px;
+      margin-bottom: 1rem;
+    }
+
+    .text.user-field {
+      width: 8rem;
+      display: inline-block;
+      margin-bottom: 0;
+    }
+    .user-info {
+      font-weight: bold;
+      display: inline-block;
+    }
+    .card {
+      height: 144px;
+      width: 540px;
+      border-radius: 4px;
+      background-color: #FFFFFF;
+      box-shadow: 0 2px 5px 0 rgba(0,0,0,0.26), 0 2px 10px 0 rgba(0,0,0,0.16);
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      margin-bottom: 1rem;
+      cursor: pointer;
+      text-decoration: none;
+    }
+
+    div.card {
+      cursor: default;
+    }
+
+    .bullet-img {
+      height: 31px;
+      width: 31px;
+    }
+
+    .card-text {
+      width: 194px;
+      color: #302C71;
+      font-size: 16px;
+      font-weight: 300;
+      line-height: 19px;
+    }
+
+    .step-img {
+      height: 120px;
+      width: 250px;
+    }
+    .link {
+      display: inline;
+      color: #2691D0;
+    }
+    .button {
+      height: 52px;
+      width: 178px;
+      border-radius: 5px;
+      background-color: #302C71;
+      font-size: 16px;
+      font-weight: 500;
+      line-height: 19px;
+      color: #FFFFFF;
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      text-transform: uppercase;
+      text-decoration: none;
+    }
+  </style>
+</head>
+<body>
+  <img src="assets/logo-registration-non-signed-in.svg">
+  <div class="welcome-message">Welcome to the All of Us Researcher Workbench!</div>
+  <div><div class="text user-field">Your User Name: </div><div class="user-info">{{USERNAME-INPUT}}</div></div>
+  <div><div class="text user-field">Your Password: </div><div class="user-info">{{PASSWORD-INPUT}}</div></div>
+  <br>
+  <div class="text">Quick guide on how to get started:</div>
+  <a class="card" href="https://accounts.google.com/signin/oauth/oauthchooseaccount?hd=fake-research-aou.org&client_id=602460048110-5uk3vds3igc9qo0luevroc2uc3okgbkt.apps.googleusercontent.com&as=1ivZqDJMGPEMvWasjAquSQ&destination=http%3A%2F%2Flocalhost%3A4200&approval_state=!ChRqd2tjYnk1b040d3VUMzNpZXBpQRIfczlfc0R4d2dERHdUQU9FaERQUmpBazFLYU13VFBSWQ%E2%88%99AB8iHBUAAAAAWxgpw3y6ICN3aJ_etNsSGfEmZbFNYV2d&xsrfsig=AHgIfE85mPI6c5DfCze82tpt7pbw8-XBRg&flowName=GeneralOAuthFlow">
+    <img class="bullet-img" src="assets/Bullet1@2x.png">
+    <div class="card-text"><div class="link">Sign into google</div> with your new account</div>
+    <img class="step-img" src="assets/Step1@2x.png">
+  </a>
+  <div class="card">
+    <img class="bullet-img" src="assets/Bullet2@2x.png">
+    <div class="card-text">Google will then prompt you to change your password</div>
+    <img class="step-img" src="assets/Step2@2x.png">
+  </div>
+  <a class="card" href="https://myaccount.google.com/security">
```

Comment:
Didn't realize you could link directly there. I kept struggling to find this link because if you open 2sv from myaccount.google.com/security, it comes with a whole bunch of baggage/isn't clear. Good to know

Reply:
That is great - I could not find that link either.